src/ComputerVisionSystem/CVS.cpp

```diff
@@ -13,14 +13,15 @@ timur::CVS::CVS(float arucoSqureDimension, int cointOfMarkers, int markerSize,
         ,_vid(cameraIndex)
         ,_camera(calibrationFileName)
 {
-    if (!_vid.isOpened())
-    {
-        throw std::exception();
-    }
+    _isCorrect = _vid.isOpened();
```

Comment:
Change name of field into more information, like "cameraIsOpen" or somth like this.